src/openlcb/MemoryConfigClient.cxxtest

```diff
@@ -289,6 +638,25 @@ TEST_F(MemoryConfigClientTest, unfreeze)
     ASSERT_TRUE(b->data()->done.is_done());
 }
 
+TEST_F(MemoryConfigClientTest, WriteNoReplyPending)
```

Comment:
@balazsracz Please review this carefully.

Reply:
looks good